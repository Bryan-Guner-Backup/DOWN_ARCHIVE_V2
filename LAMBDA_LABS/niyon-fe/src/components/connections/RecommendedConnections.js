import React from 'react'
import Paper from '@material-ui/core/Paper'

function RecommendedConnections (props) {
  return (

    <div className="rec-connect" data-test="connections">
      <div data-test="second-wrapper">
        <Paper className="paper" elevation={1}>
          <h1 className="container-header" data-test="my-connections">
          Recommended Connections
          </h1>
          <p className="container-body">Cards go here</p>
        </Paper>
      </div>
    </div>
  )
}

export default RecommendedConnections
