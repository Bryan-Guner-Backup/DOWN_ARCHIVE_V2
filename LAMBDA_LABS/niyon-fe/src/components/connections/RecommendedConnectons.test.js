import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import RecommendedConnections from './RecommendedConnections'

describe('<RecommendedConnections /> component', () => {
  let component
  beforeEach(() => {
    component = shallow(<RecommendedConnections />)
  })

  it('should match the snapshot test', () => {
    expect(toJSON(component)).toMatchSnapshot()
  })

  it('should render the component correctly', () => {
    expect(component.exists()).toBe(true)
  })
})
