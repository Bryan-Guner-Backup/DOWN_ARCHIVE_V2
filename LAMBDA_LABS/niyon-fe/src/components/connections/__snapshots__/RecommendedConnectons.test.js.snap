// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RecommendedConnections /> component should match the snapshot test 1`] = `
<div
  className="rec-connect"
  data-test="connections"
>
  <div
    data-test="second-wrapper"
  >
    <WithStyles(ForwardRef(Paper))
      className="paper"
      elevation={1}
    >
      <h1
        className="container-header"
        data-test="my-connections"
      >
        Recommended Connections
      </h1>
      <p
        className="container-body"
      >
        Cards go here
      </p>
    </WithStyles(ForwardRef(Paper))>
  </div>
</div>
`;
